app/controllers/customer_resources_controller.rb

```diff
@@ -0,0 +1,26 @@
+class CustomerResourcesController < ApplicationController
+  before_action :configure_resource
+
+  def show
+    begin
+      vendor_id, package_id, title_id = params[:id].split('-')
+      customer_resource = CustomerResource.find(
+        vendor_id: vendor_id,
+        package_id: package_id,
+        title_id: title_id
+      )
+      render jsonapi: customer_resource,
+             include: params[:include]
+    rescue Flexirest::HTTPClientException, Flexirest::HTTPServerException, Flexirest::HTTPNotFoundClientException => e
```

Comment:
this begin/rescue is cumbersome and pops up all over the place.  best thing i could think of is to implement methods in the superclass that wrap a `yield`, but that's a little heavy so I didn't want to do it without fielding other opinions.